Stripe/StripeiOSTests/CustomerAdapterTests.swift

```diff
@@ -43,8 +45,6 @@ class CustomerAdapterTests: APIStubbedTestCase {
         expectedCount: Int,
         apiClient: STPAPIClient
     ) {
-        let exp = expectation(description: "listPaymentMethod")
-        exp.expectedFulfillmentCount = expectedCount
```

Comment:
What was the reason for removing expected count?  If we decide to remove it, then we should remove it from the function parameter.

Reply:
I can't remember why we needed this, but i think you're right in that it isn't needed. Lets remove the `expectedCount` parameter and then happy to ship it.

Reply:
Oops, removed.